addons/mail/static/src/components/call_main_view/call_main_view.js

```diff
@@ -17,6 +17,18 @@ export class CallMainView extends Component {
         useComponentToModel({ fieldName: 'component' });
         useRefToModel({ fieldName: 'tileContainerRef', refName: 'tileContainer', });
         useUpdateToModel({ methodName: 'onComponentUpdate' });
+
+        onMounted(() => {
+            this.resizeObserver = new ResizeObserver(() => {
+                this.callMainView.onComponentUpdate();
+            });
+            this.resizeObserver.observe(this.root.el);
+        });
+        onWillUnmount(() => {
+            if (this.resizeObserver) {
```

Comment:
Is it actually possibly to go inside `onWillUnmount` without having gone into `onMounted`?